pkg/cmd/controller/config/config.go

```diff
@@ -0,0 +1,51 @@
+/*
```

Comment:
followup: seems like you should remove this header from the files.

Reply:
I will remove all license headers from files in a follow-up PR.

Reply:
Created issue #17.